homework/index.js

```diff
@@ -29,8 +29,42 @@
     return elem;
   }
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+  function renderRepoDetails(repo) {
```

Comment:
This is a rather large function. It is difficult to understand what is happening here. It would be better if you break this function up into smaller function, each with a descriptive name to indicate what it is doing.

Reply:
I did my best in UpDate.